app/webpacker/styles/vendor/_nhs_tables.scss

```diff
@@ -0,0 +1,170 @@
+// .nhsuk-table__heading-tab {
+//   @include heading-label($color_nhsuk-blue, $color_nhsuk-white);
+// }
+
+/* Responsive table
+========================================================================== */
+
+/**
+ * 1. Hiding the thead on mobile
+ * 2. Displaying the thead on desktop
+ * 3. Removing default screen reader behaviour
+ * 4. Assigning role of table-row on desktop to give default screen reader behaviour
+ * 5. Using justify content to space out elements in the row on mobile
+ * 6. Assigning a minimum width in case of black cell
+ * 7. Aligning content to the right on mobile
+ * 8. Aligning mobile header to left to split it from the data
+ * 9. Hiding mobile specific header from desktop view
+ * 10. Adding a display block value due to IE 11 not having full flex support
+ */
+
+.nhsuk-table-responsive {
+  margin-bottom: 0;
+  width: 100%;
+
+  thead {
+    // @include visually-hidden; /* [1] */
+
+    // @include mq($from: desktop) {
+    //   @include visually-shown(table-header-group); /* [2] */
+    // }
+
+    @include govuk-media-query($until: desktop) {
+      @include govuk-visually-hidden;
+    }
+  }
+
+  .nhsuk-table__body {
+    .nhsuk-table-responsive__heading {
+      // font-weight: $nhsuk-font-bold;
+      font-weight: bold;
+      // padding-right: nhsuk-spacing(3);
+      padding-right: govuk-spacing(3);
+      text-align: left; /* [8] */
+
+      // @include mq($from: desktop) {
+      @include govuk-media-query($from: desktop) {
+        display: none; /* [9] */
+      }
+    }
+
+    .nhsuk-table__row {
+      display: block; /* [3] */
+      // margin-bottom: nhsuk-spacing(4);
+      margin-bottom: govuk-spacing(4);
+
+      &:last-child {
+        margin-bottom: 0;
+      }
+
+      // @include mq($from: desktop) {
+      @include govuk-media-query($from: desktop) {
+        display: table-row; /* [4] */
+      }
+
+      th {
+        text-align: right;
+        // @include mq($from: desktop) {
+        @include govuk-media-query($from: desktop) {
+          text-align: left;
+        }
+      }
+
+      td {
+        display: block; // For browsers that don't support flexbox
+        display: flex;
+        justify-content: space-between; /* [5] */
+        min-width: 1px; /* [6] */
+        @media all and (-ms-high-contrast: none) { /* [10] */
+          display: block;
+        }
+
+        // @include mq($from: desktop) {
+        @include govuk-media-query($from: desktop) {
+          display: table-cell;
+        }
+
+        // @include mq($until: desktop) {
+        @include govuk-media-query($until: desktop) {
+          padding-right: 0;
+          text-align: right; /* [7] */
+          &:last-child {
+            border-bottom: 3px solid $govuk-border-colour;
+          }
+        }
+      }
+    }
+  }
+}
+
+/* Numeric tables
+========================================================================== */
+
+/**
+ * Right aligns table cells for numeric tables.
+ */
+
+.nhsuk-table__header--numeric,
```

Comment:
Is this used? Should it be?

Reply:
nope, but we're using the govuk class to do the same thing. I wanted to change this file as little as possible which is why there's a few things in here that aren't used.